apps/client/src/app/core/data/localized-lazy-data-service.spec.ts

```diff
@@ -17,7 +19,7 @@ const languages = ['en', 'fr', 'ja', 'de', 'ko', 'zh'] as const; // Not testing
  * Note that this must remain separate from the other load function, as the path names must be statically analyzable for webpack to include the resources in the bundle.
  * @param key The lazy data key to load data for.
  */
-const load = async (key: LazyDataKey) => {
+const loadStd = async (key: LazyDataKey) => {
   return await import('../../../assets/data/' + lazyFilesList[key].fileName);
 };
```

Comment:
This is going to be a big issue, because it defeats the purpose of lazy files. I originally created this system because large jsons were imported as-is and they were making the build take way too much RAM, which broke the build process inside circleci.

Lazy Data aims at providing a lighter bundle and faster build times by excluding json files from the build process and loading them at runtime from the assets instead.

This change it probably the reason why it breaks inside the CI as of now.

Reply:
Does this apply to the unit test files as well? I can't say I know enough about how the bundler works in the test runner to know for sure. 
The motivation for this was that I didn't want to rely on the same mechanisms the app relied on to load the data, as it might increase the possibility of false negatives.

Reply:
I'm not sure neither, but your circleci build fails because of an oom error, so I guess it is possible somehow oO

Reply:
Interesting! The test stage passed, but the build itself failed (and these .spec.ts files shouldn't be included in the build). I'll look into it.

Reply:
The build did not time out this time, for no discernable reason. 💁‍♀️